OracleUnifiedDirectory/samples/README.md

```diff
@@ -0,0 +1,1347 @@
+# Creating Oracle Unified Directory Docker Containers
```

Comment:
Please remove the word "Docker" as much as possible from this file. These are container images and they will build and run on any OCI image-spec and runtime-spec compliant platform.

Reply:
Taken care of this at lot of the instances. Remaining instances are to be updated through future PR.